### ISSUE TITLE
cifar100

### ISSUE THREAD

**Issue by JJJTF** [Ugenteraan/Deep_Hierarchical_Classification#3]
In the paper, the cifar100 coarse acc is 92.21% and fine acc 75.91%, but the coarse accuracy I ran out is 71%, and the fine accuracy is 58%.
Why?

**Comment by Ugenteraan** [comment]
Sorry for the late response @JJJTF. I can't give you an answer without knowing the hyperparameters you used and the logs from your training session. You could probably try tuning the hyperparameters and make sure the training/testing images are not corrupted for better accuracy. Also, when you stopped the training, was the accuracy still improving or was it already stagnant for the past few epochs?
